Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda3:4.9.2
+
+RUN conda create -n perf-dev -c conda-forge \
```

Comment:
Looking at this again, we can just install directly into the `base` conda environment. Will push a small update to do that

Reply:
Hmm I actually ran into some unexpected compilation issues (when cythonizing `distributed`) when using the `base` env. Since installing into a fresh conda env seems to work, let's stick with that

Reply:
Yeah a separate env would be my preference as well